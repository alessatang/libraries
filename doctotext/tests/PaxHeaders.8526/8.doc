30 atime=1389151478.326866086
30 ctime=1389151478.326866086
