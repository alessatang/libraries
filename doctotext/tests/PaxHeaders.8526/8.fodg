30 mtime=1384978086.707497426
30 atime=1389151478.327866082
30 ctime=1389151478.327866082
