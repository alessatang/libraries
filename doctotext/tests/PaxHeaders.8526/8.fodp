30 mtime=1384978086.710497427
30 atime=1389151478.330866069
30 ctime=1389151478.330866069
