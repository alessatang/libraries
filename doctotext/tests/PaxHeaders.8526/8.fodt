30 mtime=1384978086.713497428
30 atime=1389151478.331866065
30 ctime=1389151478.331866065
