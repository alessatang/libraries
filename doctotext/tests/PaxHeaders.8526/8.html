30 atime=1389151478.331866065
30 ctime=1389151478.331866065
