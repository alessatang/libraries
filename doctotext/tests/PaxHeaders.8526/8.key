30 mtime=1379552763.702958411
30 atime=1389151478.341866023
30 ctime=1389151478.341866023
