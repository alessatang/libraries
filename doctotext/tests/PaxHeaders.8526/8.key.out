30 mtime=1379553523.230588132
30 atime=1389151478.341866023
30 ctime=1389151478.341866023
